client/components/info-popover/style.scss

```diff
@@ -14,6 +14,8 @@
 }
 
 .popover.info-popover__tooltip {
+	outline: none;
```

Comment:
This does solve the issue and the outline is no longer displaying. 👍🏼 

I am just wondering how (if) this change could affect accessibility and whether we should apply on the component level instead of overriding the style in the Subscriptions Management. 🤔

Reply:
Related discussion: p1688469952373849-slack-C02TCEHP3HA.

→ We decided to get rid of the outline.